src/main/java/Chess/domain/ChessBoard.java

```diff
@@ -58,10 +54,10 @@ private void initializeChessboard() {
 
     private void initializeChessUnit() throws Exception {
         for (int i = 0; i < CHESSBOARD_COLUMN; i++) {
-            setUnitFromCell(BLACK_SPECIAL_CHESS_UNIT_ROW, i, INITIAL_SPECIAL_CHESS_UNIT_POSITION.get(i).getConstructor(ChessUnitColor.class).newInstance(ChessUnitColor.BLACK));
-            setUnitFromCell(BLACK_PAWN_CHESS_UNIT_ROW, i, new Pawn(ChessUnitColor.BLACK));
-            setUnitFromCell(WHITE_SPECIAL_CHESS_UNIT_ROW, i, INITIAL_SPECIAL_CHESS_UNIT_POSITION.get(i).getConstructor(ChessUnitColor.class).newInstance(ChessUnitColor.WHITE));
-            setUnitFromCell(WHITE_PAWN_CHESS_INIT_ROW, i, new Pawn(ChessUnitColor.WHITE));
+            setUnitFromCell(BLACK_SPECIAL_CHESS_UNIT_ROW, i, INITIAL_SPECIAL_CHESS_UNIT_POSITION.get(i).getConstructor(UnitColor.class).newInstance(UnitColor.BLACK));
```

Comment:
한 줄에 점(.) 이 여러 개 들어가지 않게 합시다.
```suggestion
            setUnitFromCell(
                BLACK_SPECIAL_CHESS_UNIT_ROW, i, INITIAL_SPECIAL_CHESS_UNIT_POSITION
                .get(i)
                .getConstructor(UnitColor.class)
                .newInstance(UnitColor.BLACK)
            );
```
이렇게 하면 가독성에도 도움이 되고 디버깅 시에도 한 줄씩 메모리상의 값을 검사해볼 수 있습니다.
'디미터 법칙'에 대해 검색해 보세요.